node/resolvers/geolocation.ts

```diff
@@ -172,14 +172,13 @@ function getCountry(googleAddress) {
 /* END OF COPY-PASTED CODE FROM ADDRESS-FORM  */
 
 export const queries = {
-  reverseGeocode: async (_: any, args: any, ctx: any): Promise<any> => {
+  reverseGeocode: async (_: any, args: any, ctx: Context): Promise<any> => {
```

Comment:
```suggestion
  reverseGeocode: async (_: {}, args: any, ctx: Context): Promise<any> => {
```
First parameter type is {} because it is a root resolver.

Reply:
I actually typed it as `unknown` because it doesn't matter in this case